lbrynet/daemon/auth/client.py

```diff
@@ -100,28 +102,26 @@ def f(*args, **kwargs):
 
     @classmethod
     async def get_client(cls, key_name=None):
-        api_key_name = key_name or API_KEY_NAME
+        api_key_name = key_name or "api"
+        keyring = Keyring.load_from_disk()
 
-        pw_path = os.path.join(conf.settings['data_dir'], ".api_keys")
-        keys = load_api_keys(pw_path)
-        api_key = keys.get(api_key_name, False)
+        api_key = keyring.api_key
 
-        login_url = "http://{}:{}@{}:{}".format(api_key_name, api_key.secret, conf.settings['api_host'],
-                                                conf.settings['api_port'])
+        login_url = "http{}://{}:{}@{}:{}".format("" if not conf.settings['use_https'] else "s", api_key_name,
+                                                  api_key.secret, conf.settings['api_host'], conf.settings['api_port'])
```

Comment:
This URL building is duplicated 3 times in the PR, can it be extracted into some kind of utility function or method?